### PR TITLE
fix: Export correct options for compat.

### PR DESCRIPTION
**Pull request by kinyoklion**

### CODE REVIEW

**Comment by kinyoklion** [comment]
Went a little bit too far with the common exports.

File: packages/sdk/browser/src/common.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import { BasicLogger, BasicLoggerOptions, LDLogger } from '@launchdarkly/js-client-sdk-common';
 
 import { BrowserIdentifyOptions as LDIdentifyOptions } from './BrowserIdentifyOptions';
-import { BrowserOptions as LDOptions } from './options';
```
Comment (kinyoklion) [inline packages/sdk/browser/src/common.ts:4]: This file shouldn't export options as the compat options are a little different.

File: packages/sdk/browser/src/compat/index.ts [hunk 2]
```diff
@@ -5,13 +5,14 @@
...
-import { LDContext, LDOptions } from '@launchdarkly/js-client-sdk-common';
+import { LDContext } from '@launchdarkly/js-client-sdk-common';
 
 import { LDClient } from './LDClientCompat';
 import LDClientCompatImpl from './LDClientCompatImpl';
+import { LDOptions } from './LDCompatOptions';
 
 export * from '../common';
-export type { LDClient };
+export type { LDClient, LDOptions };
```
Comment (kinyoklion) [inline packages/sdk/browser/src/compat/index.ts:15]: Exporting the compat options.

File: packages/sdk/browser/src/index.ts [hunk 3]
```diff
@@ -13,10 +13,10 @@
 import { AutoEnvAttributes } from '@launchdarkly/js-client-sdk-common';
 
 import { BrowserClient, LDClient } from './BrowserClient';
-import { LDOptions } from './common';
+import { BrowserOptions as LDOptions } from './options';
 
 export * from './common';
-export type { LDClient };
+export type { LDClient, LDOptions };
```
Comment (kinyoklion) [inline packages/sdk/browser/src/index.ts:19]: Exporting standard browser options.

**Review by louis-launchdarkly — APPROVED** [review]

**Review by cwaldren-ld — APPROVED** [review]

**Pull request merged**